data/campaigns/Heir_To_The_Throne/scenarios/11_Elensefar_After.cfg

```diff
@@ -627,7 +629,7 @@
                     image="icons/sandals.png"
                     name=_"Seafarer’s Sandals"
                     description=_"This enchanted footwear repels nearby water, allowing
-its wearer to walk on ocean or snow as if standing on solid ground."
+its wearer to walk on sea or snow as if standing on solid soil."
```

Comment:
Solid ground sounds right to me. I don't think I've ever heard the phrase 'solid soil' before.

Reply:
Oh... I see. I completely missed the alliteration. Well, I leave it up to you, then.